packages/shared/src/components/sidebar/Sidebar.tsx

```diff
@@ -75,6 +92,79 @@ export default function Sidebar({
     return <></>;
   }
 
+  if (isNewMobileLayout && isTablet) {
+    const buttonProps: ButtonProps<'a' | 'button'> = {
+      variant: ButtonVariant.Tertiary,
+      size: ButtonSize.Large,
+      className: 'w-full',
+    };
+
+    return (
+      <SidebarAside
+        data-testid="sidebar-aside"
+        className={classNames('max-w-16 basis-10 items-center gap-4')}
```

Comment:
Just checking should this not be just a static width? (instead of using max-w)

Reply:
Also, not sure `max-w-` works with numbers ootb.